apis/kueue/webhooks/workload_webhook.go

```diff
@@ -140,6 +145,15 @@ func ValidateWorkload(obj *kueue.Workload) field.ErrorList {
 	return allErrs
 }
 
+func validatePodSetName(name string, fldPath *field.Path) field.ErrorList {
```

Comment:
Seems duplicate with `validateNameReference`.

Reply:
So far, they are exclusive to one type of resource, so not worth putting in common.